highlighters/main/main-highlighter.zsh

```diff
@@ -857,7 +857,7 @@ _zsh_highlight_main_highlighter_highlight_list()
                           fi
                           _zsh_highlight_main__stack_pop 'R' reserved-word
                         else
-                          if _zsh_highlight_main_highlighter_check_path $arg; then
+                          if _zsh_highlight_main_highlighter_check_path $arg 1; then
```

Comment:
Regresses `sudo -e /etc/passwd`: in current master is `path`, with this branch is `unknown-token`.  See #678.

Reply:
Fixed by fixing #678 on master.